srtcore/core.h

```diff
@@ -134,11 +134,9 @@ extern const SRT_SOCKOPT srt_post_opt_list [];
 enum GroupDataItem
 {
     GRPD_GROUPID,
-    GRPD_GROUPTYPE,
-    GRPD_WEIGHT,
+    GRPD_GROUPDATA,
 
-
-    GRPD__SIZE
+    GRPD_E_SIZE
```

Comment:
Why is the weight deleted from here?

Reply:
These are indexes to the `uint32_t` array. Previously it was type and weight packet to separate cells. Now all - weight, type and flags - are in a single cell, therefore it uses the index name collectively as "group data".

Reply:
Ok, makes sense.